lib/PEX.ts

```diff
@@ -72,16 +72,16 @@ export class PEX {
    */
   public evaluatePresentation(
     presentationDefinition: IPresentationDefinition,
-    presentations: OrArray<OriginalVerifiablePresentation | IPresentation>,
+    presentations: OrArray<OriginalVerifiablePresentation | IPresentation | SdJwtDecodedVerifiableCredentialWithKbJwtInput>,
```

Comment:
I think we use the `OriginalVerifiablePresentation` for SD-JWT (so always pass the SD-JWT in it's encoded (string) variant). I think this is maybe where the type conflicts come from?

Reply:
We revised this to use updated types from ssi-types